keyboards/kbdfans/kbd67/mkiirgb/keymaps/dnsnrk/keymap.c

```diff
@@ -0,0 +1,95 @@
+/* Copyright 2019 dnsnrk 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define RC_ENT RCTL_T(KC_ENT)
+#define LC_CPS LCTL_T(KC_CAPS)
+#define KC_VDN KC__VOLDOWN
+#define KC_VUP KC__VOLUP
```

Comment:
Should be able to use `KC_VOLU` and `KC_VOLD` here, as long as `EXTRAKEYS` is enabled. 
https://docs.qmk.fm/#/keycodes_basic?id=media-keys

Reply:
yep, they work equally well. Supposedly, I used the first codes appearing in the documentation without further scrolling down.